lib/matplotlib/ticker.py

```diff
@@ -2952,9 +2952,9 @@ def __call__(self):
             vmin, vmax = vmax, vmin
 
         t0 = majorlocs[0]
-        tmin = ((vmin - t0) // minorstep + 1) * minorstep
-        tmax = ((vmax - t0) // minorstep + 1) * minorstep
-        locs = np.arange(tmin, tmax, minorstep) + t0
+        tmin = round((vmin - t0) / minorstep)
```

Comment:
I think this needs an API change note.  /doc/api/next_api_changes/behavior is the correct place for this.